mubench.pipeline/data/detector_execution.py

```diff
@@ -21,34 +17,17 @@ class Result(Enum):
     timeout = 2
 
 
-class DetectorMode(IntEnum):
-    mine_and_detect = 0
-    detect_only = 1
-
-
 class DetectorExecution:
```

Comment:
I felt like running and executing is basically the same thing, so I didn't bother renaming it.

Reply:
I see you point... I'm trying to be consistent, because it's not necessarily obvious that it's the same. Since we just renamed the CLI task to `run` and have the `run.yml` already, I think I prefer using `run` in the code as well.